<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrthoQualityWindow</class>
 <widget class="QMainWindow" name="OrthoQualityWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ortho-image Quality Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/resource.qrc">
    <normaloff>:/icon/WindowIcon.png</normaloff>:/icon/WindowIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowIcon">
    <iconset>
     <normaloff/>
    </iconset>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Ortho-image Quality Tool</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Input type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox1">
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Point cloud (without index)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Stereoplotter features</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="loadButton">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Load testing points</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resource/resource.qrc">
             <normaloff>:/icon/fileopen.png</normaloff>:/icon/fileopen.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="deleteButton">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Delete point</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resource/resource.qrc">
             <normaloff>:/icon/remove.png</normaloff>:/icon/remove.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>Show terrain points</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="calculateButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Calculate quality</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resource/resource.qrc">
           <normaloff>:/icon/calculate.png</normaloff>:/icon/calculate.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="saveQButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Save ortho-image quality report</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resource/resource.qrc">
           <normaloff>:/icon/filesave</normaloff>:/icon/filesave</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="loadQButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Open ortho-image quality report</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resource/resource.qrc">
           <normaloff>:/icon/fileopen.png</normaloff>:/icon/fileopen.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolButton" name="doneButton">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Exit</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resource/resource.qrc">
             <normaloff>:/icon/done.png</normaloff>:/icon/done.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTableWidget" name="tableWidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>192</height>
        </size>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>140</number>
       </attribute>
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>3</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>4</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>5</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Terrain X</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Terrain Y</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ortho-image X</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ortho-image Y</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>X error</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Y error</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Planimetric Error</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/find.png</normaloff>:/icon/find.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/exit2.png</normaloff>:/icon/exit2.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionView_report">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/table.png</normaloff>:/icon/table.png</iconset>
   </property>
   <property name="text">
    <string>View &amp;report</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionFit_view">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/fit.png</normaloff>:/icon/fit.png</iconset>
   </property>
   <property name="text">
    <string>Fit &amp;view</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSet_mark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icon/mark.png</normaloff>:/icon/mark.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Set mark</string>
   </property>
   <property name="iconText">
    <string>Set mark</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionInterior_orientation">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/interior.ico</normaloff>:/icon/interior.ico</iconset>
   </property>
   <property name="text">
    <string>Interior &amp;orientation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/move.png</normaloff>:/icon/move.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Move</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionActive_grid">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/grid.png</normaloff>:/icon/grid.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Active grid</string>
   </property>
   <property name="iconText">
    <string>Active grid</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
